go/libkb/session.go

```diff
@@ -308,6 +308,8 @@ func (s *Session) Invalidate() {
 	s.token = ""
 	s.csrf = ""
 	s.checked = false
+	// XXX this might be a mistake as the session might be refreshed
+	// but it was here before, so...
```

Comment:
Hmm, maybe we should get rid of it?  We sometimes get random "logout bugs", maybe this is the cause?